test/rules/super_goes_last.dart

```diff
@@ -33,9 +32,7 @@ class C extends A {
 
 ''', [
       error(HintCode.UNUSED_FIELD, 61, 2),
-      // Update to `SUPER_INVOCATION_NOT_LAST` when analyzer 2.8.0+ is published
-      // https://github.com/dart-lang/linter/issues/3069
-      // error(CompileTimeErrorCode.INVALID_SUPER_INVOCATION, 84, 5),
+      error(CompileTimeErrorCode.SUPER_INVOCATION_NOT_LAST, 84, 5),
```

Comment:
Was this suppose to replace line 36? If both are currently being supported, should we stop reporting the lint when the error is being reported?

Reply:
I think the real answer is we remove the lint.  It's been deprecated for a while...

See also: https://github.com/dart-lang/linter/issues/1368#issuecomment-455253837